apps/web-giddh/src/app/subscription/subscription-list/subscription-list.component.html

```diff
@@ -0,0 +1,383 @@
+                            {{ element.billingAccount?.name ? element.billingAccount?.name : "-" }}
+                        </td>
+                    </ng-container>
+                    <ng-container matColumnDef="subscriberName">
+                        <th
+                            mat-header-cell
+                            *matHeaderCellDef
+                            class="td-search-box"
+                            #searchSubscriberContainer
+                            (clickOutside)="
+                                handleClickOutside($event, searchSubscriberContainer, localeData?.subscriber)
+                            "
+                        >
+                            <ng-container
+                                *ngTemplateOutlet="
+                                    searchTemplate;
+                                    context: {
+                                        $implicit: showSubscriber,
+                                        fieldName: localeData?.subscriber,
+                                        formControl: subscriptionListForm.controls['subscriberName'],
+                                        title: localeData?.subscriber
+                                    }
+                                "
+                            ></ng-container>
+                        </th>
+                        <td class="subscription-td" mat-cell *matCellDef="let element">
+                            {{ element.subscriber?.name ? element.subscriber?.name : "-" }}
+                        </td>
+                    </ng-container>
+                    <ng-container matColumnDef="countryName">
+                        <th
+                            mat-header-cell
+                            *matHeaderCellDef
+                            class="td-search-box"
+                            #searchCountryContainer
+                            (clickOutside)="
+                                handleClickOutside($event, searchCountryContainer, commonLocaleData?.app_country)
+                            "
+                        >
+                            <ng-container
+                                *ngTemplateOutlet="
+                                    searchTemplate;
+                                    context: {
+                                        $implicit: showCountry,
+                                        fieldName: commonLocaleData?.app_country,
+                                        formControl: subscriptionListForm.controls['countryName'],
+                                        title: commonLocaleData?.app_country
+                                    }
+                                "
+                            ></ng-container>
+                        </th>
+                        <td class="subscription-td" mat-cell *matCellDef="let element">
+                            {{ element.country?.countryName ? element.country?.countryName : "-" }}
```

Comment:
```suggestion
                            {{ element.country?.countryName ?? "-" }}
```
do this everywhere n this file